tests/phpunit/tests/rest-api/rest-server.php

```diff
@@ -931,6 +931,37 @@ public function test_link_embedding_without_links() {
 		$this->assertSame( 'data', $result['untouched'] );
 	}
 
+	/**
+	 * Ensure embedding is with links in the data.
```

Comment:
```suggestion
	 * Ensure embedding is with links in the data.
	 *
	 * @ticket 43439
```

Ticket number is missing.